src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -42,7 +53,8 @@
 import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
 
 // Components
-import CorrectNameRequest from './CorrectNameRequest.vue'
+import CorrectNameRequest from '@/components/Company/CompanyName/CorrectNameRequest.vue'
+import CorrectCompanyName from '@/components/Company/CompanyName/CorrectCompanyName.vue'
```

Comment:
If you use an index file then you could import both components in 1 line.

Reply:
Due to the index file, the file where we import and having the files exporting all from the same directory was actually causing some weirdness here.

Reply:
> Looks really good overall.
> 
> I like the component break-down. It makes for smaller/simpler components that do only specific things.
> 
> What about ... unit tests?

This is a great question, i feel like these tickets were a little under-estimated due to being our first correction within a new Edit-UI. That being said, i've been playing catch up and have put them off. Are we ok to create a tech debt ticket, bring it in for next spring (if approved) and i can certainly explain why to the team.

Reply:
OK. If I remember then I will have a look at this later.